### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [HuyTranTuan/Week1_EcommercePurchase#1]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You actually can go through every questions. This is a great start :D
- You can make use of indexing and applying filtration well.
- You imported and utilized new libraries. Awesome !

## Things to work on:
- You should make use of Pandas and reference its documentation more, Pandas is powerful, and it can make coding much easier.

## To sum up
- Your work is excellent and it shows that you can code very well.
- You are demonstrating that you are understanding the material and doing a great job of applying it. Keep it up!
